docs/serving/samples/hello-world/helloworld-python/index.md

```diff
@@ -5,4 +5,230 @@ weight: 1
 type: "docs"
 ---
 
-{{% readfile file="README.md" %}}
```

Comment:
I guess I'm lacking context: Is the readfile destroying the tabs?

Reply:
yeah, the readfile that includes things doesn’t work with the “short codes” for tab. You can find more details here -> https://knative.slack.com/archives/CE4MVFVAQ/p1600286984016100